flexport/latitude
# FilterButton styles no longer flash-transition on mount.

## Description:

Currently, when FilterButton is mounted, there is a brief 150ms when the styles on the FilterButton transition from its inherited styles to its self defined styles. This is due to the way Aphrodite applies styles -- Aphrodite styles are injected onto an element after the element is mounted. Since FilterButton's styles contains transition properties, when the styles are injected a transition will occur from the element's inherited properties to its Aphrodite styles.

This PR fixes this behavior by applying the transition styles only after Aphrodite has finished injecting its styles.

## Manual Test Plan:
Go to storybook filters example on master, observe there is a flash transition on page refresh.
Check out this branch, go to storybook, observe there is no longer a flash-transition

## Screenshot:

before behavior (slowed for emphasis):
![filter-refresh](https://user-images.githubusercontent.com/5897853/70762764-ae547000-1d06-11ea-8edc-25b08e118e28.gif)
